horizon/backend/providers/auth/ldap.py

```diff
@@ -9,10 +9,10 @@
 Similar to:
 * `JupyterHub LDAPAuthenticator <https://github.com/jupyterhub/ldapauthenticator>`_
 * `Flask-AppBuilder LDAP integration <https://github.com/dpgaspar/Flask-AppBuilder/blob/master/docs/config.rst>`_, used in Apache Airflow
-"""
+"""  # noqa: E501
 
 import logging
-from asyncio import TimeoutError
+from asyncio import TimeoutError  # noqa: A004
```

Comment:
It's just an alias for builtin TimeoutError, this line can be removed
https://docs.python.org/3/library/asyncio-exceptions.html#asyncio.TimeoutError